tomcat/src/main/java/camp/nextstep/request/Cookie.java

```diff
@@ -0,0 +1,23 @@
+package camp.nextstep.request;
+
+import java.util.UUID;
+
+public class Cookie {
+    public static final String JSESSIONID_NAME = "JSESSIONID";
+
+    public static String randomJsessionId() {
+        return UUID.randomUUID().toString();
+    }
+
+    private final String key;
+    private final String value;
+
+    public Cookie(String key, String value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
```suggestion
    private final String key;
    private final String value;

    public Cookie(String key, String value) {
        this.key = key;
        this.value = value;
    }
    
    public static String randomJsessionId() {
        return UUID.randomUUID().toString();
    }

    public String getValue() {
        return value;
    }
```

클래스 내부 선언은 상태값 - 생성자 - 메서드 순으로 선언하는것이 컨벤션 입니다 😄

Reply:
깔끔해진 것 같습니다. 반영했습니다. :)